src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/help/Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry.md

```diff
@@ -0,0 +1,215 @@
+---
+external help file: Microsoft.Azure.Commands.DataLakeAnalytics.dll-Help.xml
+Module Name: AzureRM.DataLakeAnalytics
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.datalakeanalytics/get-azurermdatalakeanalyticscatalogitemaclentry
+schema: 2.0.0
+---
+
+# Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry
+
+## SYNOPSIS
+Gets an entry in the ACL of a catalog or catalog item in Data Lake Analytics.
+
+## SYNTAX
+
+### GetCatalogAclEntry (Default)
+```
+Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry [-Account] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### GetCatalogAclEntryForUserOwner
+```
+Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry [-Account] <String> [-UserOwner]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### GetCatalogAclEntryForGroupOwner
+```
+Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry [-Account] <String> [-GroupOwner]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### GetCatalogItemAclEntry
+```
+Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry [-Account] <String> [-ItemType] <String>
+ [-Path] <CatalogPathInstance> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### GetCatalogItemAclEntryForUserOwner
+```
+Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry [-Account] <String> [-UserOwner] [-ItemType] <String>
+ [-Path] <CatalogPathInstance> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### GetCatalogItemAclEntryForGroupOwner
+```
+Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry [-Account] <String> [-GroupOwner] [-ItemType] <String>
+ [-Path] <CatalogPathInstance> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry** cmdlet gets a list of entries (ACEs) in the access control list (ACL) of a catalog or catalog item in Data Lake Analytics.
+
+## EXAMPLES
+
+### Example 1: Get the ACL for a catalog
+```powershell
+PS C:\> Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry -Account "contosoadla"
```

Comment:
@idear1203 for the examples you're adding in this PR, would you be able to add the corresponding output from the cmdlet call so users have an idea of what is returned?

Reply:
Output are added for examples of `Get-AzureRmDataLakeAnalyticsCatalogItemAclEntry` and `Set-AzureRmDataLakeAnalyticsCatalogItemAclEntry`

---
In reply to: [197487475](https://github.com/Azure/azure-powershell/pull/6252#discussion_r197487475) [](ancestors = 197487475)